Makefile

```diff
@@ -9,4 +9,4 @@ clean:
 	cd build && $(MAKE) clean
 
 degen:
-	rm build/* -rf
+	rm build/* -rf
```

Comment:
git gets a bit mad if you remove newlines :c

Reply:
I'm adding a blank line at the bottom, if that's what you mean.